website/static/js/pointers.js

```diff
@@ -13,196 +13,279 @@ var oop = require('js/oop');
-                self.searchAllProjectsSubmitText(SEARCH_ALL_SUBMIT_TEXT);
-                self.searchMyProjectsSubmitText(SEARCH_MY_PROJECTS_SUBMIT_TEXT);
-                self.loadingResults(false);
+                nodes.forEach(function(each){
+                    if (each.type === 'registrations'){
+                        each.dateRegistered = new osfHelpers.FormattableDate(each.attributes.date_registered);
+                    } else {
+                        each.dateCreated = new osfHelpers.FormattableDate(each.attributes.date_created);
+                        each.dateModified = new osfHelpers.FormattableDate(each.attributes.date_modified);
+                    }
+                    if (embedNodeIds.indexOf(each.id) !== -1){
+                        self.selection.push(each);
+                    }
+                    count -= 1;
+                    if (count === 0){
+                        self.results(nodes);
+                        self.currentPage(self.pageToGet());
+                        self.numberOfPages(Math.ceil(response.links.meta.total / response.links.meta.per_page));
+                        self.addNewPaginators();
+                    }
+                });
+                self.doneSearching();
             });
-        } else {
-            self.results([]);
-            self.currentPage(0);
-            self.totalPages(0);
-            self.searchAllProjectsSubmitText(SEARCH_ALL_SUBMIT_TEXT);
-            self.searchMyProjectsSubmitText(SEARCH_MY_PROJECTS_SUBMIT_TEXT);
-        }
-    },
-    addTips: function(elements, data) {
-        elements.forEach(function(element) {
-            var titleText = '';
-            if (data.isRegistration) {
-                titleText = 'Registered: ' + data.dateRegistered.local;
-            } else {
-                titleText = 'Created: ' + data.dateCreated.local + '\nModified: ' + data.dateModified.local;
-            }
-            $(element).tooltip({
-                title: titleText
+            requestNodeLinks.fail(function(xhr){
+                self.searchWarningMsg(xhr.responseJSON && xhr.responseJSON.message_long);
+                count -= 1;
+                if (count === 0){
+                    self.results(nodes);
+                    self.currentPage(self.pageToGet());
+                    self.numberOfPages(Math.ceil(response.links.meta.total / response.links.meta.per_page));
+                    self.addNewPaginators();
+                }
+                self.doneSearching();
             });
         });
+        requestNodes.fail(function(xhr){
+            self.searchWarningMsg(xhr.responseJSON && xhr.responseJSON.message_long);
+            self.doneSearching();
+        });
     },
-    add: function(data) {
-        this.selection.push(data);
-        // Hack: Hide and refresh tooltips
-        $('.tooltip').hide();
-        $('.pointer-row').tooltip();
-    },
-    remove: function(data) {
+    add: function(data){
```

Comment:
Button mashing add/remove causes things to break
<img width="666" alt="screen shot 2017-07-18 at 11 00 59 am" src="https://user-images.githubusercontent.com/1322421/28324040-7c124d40-6ba8-11e7-83f3-785426d765f1.png">